packages/select/src/ListMenu.tsx

```diff
@@ -99,9 +107,11 @@ const ListMenu = React.forwardRef<HTMLUListElement, ListMenuProps>(
         active={open && !disabled}
         spacing={6}
         align={Align.Bottom}
-        justify={Justify.Middle}
+        justify={dropdownAutoWidth ? Justify.Start : Justify.Middle}
         adjustOnMutation
-        className={className}
+        className={cx(popoverClassName, className, {
+          [autoWidthStyles]: dropdownAutoWidth && !usePortal,
```

Comment:
I don't think we need the `!usePortal` check here since the width will be set to max-content by default in those cases